packages/bitcore-node/src/modules/ethereum/models/block.ts

```diff
@@ -17,6 +17,7 @@ export class EthBlockModel extends BaseBlock<IEthBlock> {
 
   async onConnect() {
     super.onConnect();
+    this.collection.createIndex({ chain: 1, network: 1, height: 1 }, { background: true });
```

Comment:
we already have 
```
chain: 1, network: 1, processed: 1, height: -1
```

Reply:
adjusted queries to to force use existing index